admin/load.php

```diff
@@ -336,3 +336,88 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Initialise Admin Pointer
+ *
+ * Handles the bootstrapping of the admin pointer.
+ * Mainly jQuery code that is self-initialising.
+ *
+ * @since 1.0.0
+ * @return void
+ */
+function perflab_admin_pointer() {
+
+	wp_enqueue_style( 'wp-pointer' );
+	wp_enqueue_script( 'wp-pointer', false, array( 'jquery' ) );
```

Comment:
Any particular reason not to load this in the footer instead?

```suggestion
	wp_enqueue_script( 'wp-pointer', false, array( 'jquery' ), true );
```

Reply:
Actually, none of these additional parameters should be present - the script is registered by WP core, so we shouldn't tamper with it, other than enqueuing it. I don't think providing parameters here even does anything, for example the script depends on `jquery` anyway so it's unnecessary to provide that dependency here again.
```suggestion
	wp_enqueue_script( 'wp-pointer' );
```

Reply:
This has been updated, args removed.